blog/docs/releases/announcing-knative-v1-6-release.md

```diff
@@ -0,0 +1,156 @@
+---
+title: "v1.6 release"
+linkTitle: "v1.6 release"
+Author: "Evan Anderson and Paul Schweigert"
+Author handle: https://github.com/dprotaso
+date: 2022-07-12
+description: "Knative 1.6 release announcement"
+type: "blog"
+---
+
```

Comment:
This can be removed now since it's no longer needed for mkdocs - see https://github.com/knative/docs/pull/5059/files for how we're formatting the date etc

Reply:
Opened an issue to update the template if it still has this: https://github.com/knative/docs/issues/5099
@evankanderson what template / file did you use to create this?

Reply:
It was a hackmd file. /shrug

Reply:
@psschwei can you please open a docs issue to remove them?